Aptoide/aptoide-client-v8
# [ASV-1913] Created new xml for appview poa element;

**What does this PR do?**

This PR aims to implement the newly designed element for poa in appview. It does not include fomo elements at this point even though on of them is already present in the xml;

**Database changed?**

No

**Where should the reviewer start?**


**How should this be manually tested?**

Since there was a small refactor to the install button you should not only test everything related to this flow (apps with poa) but also the install button cases (update, downgrade, install and migrate);

**What are the relevant tickets?**

  Tickets related to this pull-request: [ASV-1913](https://aptoide.atlassian.net/browse/ASV-1913)

**Code Review Checklist**
